### PR TITLE
fix: Several dark/light mode issues with new logo

### PR DESCRIPTION
**Pull request by iAverages**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Fixed some dark/light mode issues with the new logo. 

The displayed logo (dark/light) will now display correctly depending on the selected theme, for the logo within the nav bar, or the user's preferred color mode for the favicon.

Change also made to logo in README.md

---

## Screenshots
Before:
Before docs
Before README.md

After: 

README.md
README.md

Light preferred & light theme
Light&Light

Dark preferred & light theme
Dark&Light

Light preferred & Dark theme
Light&Dark

Dark preferred & dark theme
Dark&Dark

💯

### CODE REVIEW

File: www/src/components/navigation/navbar.astro [hunk 1]
```diff
@@ -108,3 +121,13 @@ const navbarLinks: Array<{ href: string; label: string }> = [
     window.location.pathname = "/branding";
   });
 </script>
+<style is:global>
```
Comment (c-ehrlich) [inline www/src/components/navigation/navbar.astro:124]: is it not possible to achieve this with tailwind classes?

Comment (iAverages) [reply www/src/components/navigation/navbar.astro:124]: Uh tbh you're probably right, I completely forgot tailwind was on this some how.

**Comment by c-ehrlich** [comment]
hmm not sure whats going on with the vercel deployment, ping me again if this isnt solved soon

**Comment by iAverages** [comment]
@c-ehrlich Want to try again now?

**Pull request merged**